tests/chainerx_tests/unit_tests/routines_tests/test_indexing.py

```diff
@@ -236,6 +239,9 @@ def forward_xp(self, inputs, xp):
         axis = self.axis
         indices_type = self.indices_type
         a, = inputs
+        if (xp is chainerx and self.mode == 'raise'
+                and 'cuda' in xp.get_default_device().name):
+            pytest.skip('CUDA is not supportted with mode="raise"')
```

Comment:
A nit but how about aligning the way we skip tests (https://github.com/chainer/chainer/pull/8197/files#diff-d2a5309d1c7b10cc433eb74746d256c3R220)?

Reply:
linalg tests skips in a similar way